benchmark/wildtime_benchmarks/data_generation_huffpost.py

```diff
@@ -22,7 +22,7 @@ class HuffpostDownloader(Dataset):
     time_steps = [i for i in range(2012, 2019)]
     input_dim = 44
     num_classes = 11
-    drive_id = "1jKqbfPx69EPK_fjgU9RLuExToUg7rwIY"
```

Comment:
That should not change

Reply:
Oh snap, good catch! Looks like a false positive in the typo checker